src/main/java/com/dimsirka/animalservice/services/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.dimsirka.animalservice.services.impl;
+
+import com.dimsirka.animalservice.entities.Order;
+import com.dimsirka.animalservice.exceptions.OrderNotFoundException;
+import com.dimsirka.animalservice.exceptions.OrderNotFoundException;
+import com.dimsirka.animalservice.repositories.OrderRepository;
+import com.dimsirka.animalservice.repositories.OrderRepository;
+import com.dimsirka.animalservice.services.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    private OrderRepository orderRepository;
+
+    @Autowired
+    public OrderServiceImpl(OrderRepository orderRepository) {
+
+        this.orderRepository = orderRepository;
+    }
+
+    @Override
+    public Order create(Order order) {
+
+        return orderRepository.save(order);
+    }
+
+    @Override
+    public Order update(Order order) {
+        getByIdOrThrowException(order.getId());
+        return orderRepository.save(order);
```

Comment:
Try to add logic using the existing order object to modify only desirable fields. For now allow update only order status. Follow the example below!
```
@Override
    public Order update(Order order) {
        Order persistentOrder = getByIdOrThrowException(order.getId());
        persistentOrder.setOrderStatus(order.getOrderStatus());
        return orderRepository.save(persistentOrder);
    }
```